controllers/managedclustermodule_reconciler.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"context"
+	"fmt"
+
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+
+	kmmv1beta1 "github.com/kubernetes-sigs/kernel-module-management/api/v1beta1"
+)
+
+// ManagedClusterModuleReconciler reconciles a ManagedClusterModule object
+type ManagedClusterModuleReconciler struct {
+	client.Client
+
+	Scheme *runtime.Scheme
+}
+
+//+kubebuilder:rbac:groups=kmm.sigs.k8s.io,resources=managedclustermodules,verbs=get;list;watch;create;update;patch;delete
```

Comment:
Why does the controller should be able to create `ManagedClusterModules`?

Reply:
This is a very good question. This marker was created by `kubebuilder` when I created the CRD and the reconciler, but I can't find any reason for the reconciler's `ServiceAccount` to have `create` and `delete` permissions over `ManagedClusterModule` CRs. I will remove them.